code/game/objects/items/tanks/tank_types.dm

```diff
@@ -144,7 +144,7 @@
 	w_class = WEIGHT_CLASS_SMALL
 	force = 4
 	distribute_pressure = TANK_DEFAULT_RELEASE_PRESSURE
-	volume = 1 //Tiny. Real life equivalents only have 21 breaths of oxygen in them. They're EMERGENCY tanks anyway -errorage (dangercon 2011)
+	volume = 3 //Tiny. Real life equivalents only have 21 breaths of oxygen in them. They're EMERGENCY tanks anyway -errorage (dangercon 2011)
```

Comment:
what would this be without the change?

Reply:
I get the feeling this comment in the code wasn't stating the old tanks got 21 breaths, but was emphasising how ridiculously small an e-tank is IRL. 21 breaths. That's a minute at best IRL and 3 minutes in-game with our Olypmian atheles who only breathe every 8 seconds. Our old e-tanks lasted much longer than that.